src/Benchmarks.ServerJob/Scenario.cs

```diff
@@ -13,6 +13,7 @@ public enum Scenario
     {
         Plaintext,
         Json,
+        Copy,
```

Comment:
Then `CopyToAsync` would be a better name. At least make the names consistent; don't use `post` some of the time.

Also, update `Scenarios` to move the `Copy` property above the `Scenarios below this point are not currently automated.` comment.